index.html

```diff
@@ -673,16 +673,35 @@ <h4>Steve Fifteen</h4>
             </div>
           </div>
 
+        
+            <!--machoguabednego6@gmail.com	-->
+              <div class="col-lg-3 col-md-6 d-flex align-items-stretch">
+                <div class="member" data-aos="fade-up" data-aos-delay="200">
+                  <div class="member-img">
+                    <img src="assets/img/team/Machogu.jpeg" class="img-fluid" alt="Machogu's image" width="600" height="600">
+                    <div class="social">
+                      <a href="https://twitter.com/Machogu_"><i class="bi bi-twitter"></i></a>
+                      <a href=""><i class="bi bi-facebook"></i></a>
+                      <a href=""><i class="bi bi-instagram"></i></a>
+                      <a href="https://www.linkedin.com/in/abednego-machogu-805841215/"><i class="bi bi-linkedin"></i></a>
+                    </div>
+                  </div>
+                  <div class="member-info">
+                    <h4>Machogu</h4>
+                    <span>Software Engineer</span>
+                  </div>

```

Comment:
Is this code supposed to be inside the row div from line 697 below? Ideally where you have removed your name? see suggestion below
```suggestion
```